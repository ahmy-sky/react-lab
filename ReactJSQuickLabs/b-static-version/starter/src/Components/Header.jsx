import React from 'react'
import logo from './images/qalogo.svg'

function Header() {
  return (
    <>
        <header></header>
        <nav className='navbar navbar-expand-sm'>
            <a
                href='https://www.qa.com'
                className='navbar-brand'
                target='_blank'
                rel='noreferrer'
            >
                <img
                    src={logo}
                    alt='QA Ltd'
                    width={100}
                />
            </a>
            <a className="navbar-brand" href="/">
                <h1>Todo Appliaction</h1>
            </a>
        </nav>
    </>
  )
}

export default Header