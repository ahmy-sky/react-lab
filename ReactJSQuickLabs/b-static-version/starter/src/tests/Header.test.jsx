import Header from "../Components/Header"
import React from 'react'
import { create } from 'react-test-renderer'

test('Header matches snapshot', () => { 
    const header = create(<Header />)
    expect(header.toJSON()).toMatchSnapshot()
 });