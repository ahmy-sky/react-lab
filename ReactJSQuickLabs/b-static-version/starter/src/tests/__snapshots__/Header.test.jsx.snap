// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
Array [
  <header />,
  <nav
    className="navbar navbar-expand-sm"
  >
    <a
      className="navbar-brand"
      href="https://www.qa.com"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="QA Ltd"
        src="qalogo.svg"
        width={100}
      />
    </a>
    <a
      className="navbar-brand"
      href="/"
    >
      <h1>
        Todo Appliaction
      </h1>
    </a>
  </nav>,
]
`;
